evil-collection-helm.el

```diff
@@ -165,10 +165,25 @@
     "%" 'helm-ff-run-query-replace-regexp
     "D" 'helm-ff-run-delete-file)       ; Ivy has "D".
 
+  ;; These helm bindings should always exist, the evil equivalents do
+  ;; nothing useful in the minibuffer (error or pure failure).
+  ;; RET can't do a second line in the minibuffer.
+  ;; The C-n/C-p completions error with 'helm in helm' session.
+  ;; C-o switches to evil state (again, not useful).
+  (evil-collection-define-key '(insert normal) 'helm-map
+    (kbd "RET") 'helm-maybe-exit-minibuffer
+    (kbd "M-v") 'helm-previous-page
```

Comment:
This binding was already fine by me.  I wonder why our setups are different.

Reply:
I may have forgot to test M-v, was C-v fine as well? (that's the one that I ran into - it did nothing), so I added M-v for explicit completeness